CONTRIBUTING.md

```diff
@@ -14,7 +14,8 @@ Part of the GitHub Flow is submitting pull requests. See
 https://help.github.com/articles/using-pull-requests/ for a good overview of
 what pull requests are and how to use them. Internally, we use the _shared
 repository model_ for collaborative development, though we encourage external
-contributors to contribute through the _fork & pull model_.
+contributors to contribute through the _fork & pull model_ and assign a member
+of the core team to review the pull request.
```

Comment:
We can do that, but I'm worried that it's a high barrier for external
contributors. What if they don't know all the members of the core team?

I wonder if there's some good way to manage pull requests when people
aren't assigned.

On Thu, Aug 27, 2015 at 4:15 PM, Max Wofford <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/hackedu/hackedu/pull/104#discussion_r38157408>:
>
> > @@ -14,7 +14,8 @@ Part of the GitHub Flow is submitting pull requests. See
> >  https://help.github.com/articles/using-pull-requests/ for a good overview of
> >  what pull requests are and how to use them. Internally, we use the _shared
> >  repository model_ for collaborative development, though we encourage external
> > -contributors to contribute through the _fork & pull model_.
> > +contributors to contribute through the _fork & pull model_ and assign a member
> > +of the core team to review the pull request.
>
> How about having the contributor mentioning someone in the PR description?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/104/files#r38157408>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us